test/each-test.ts

```diff
@@ -11,6 +11,8 @@ if (typeof window !== 'undefined') {
   addClosest(window);
 }
 
-afterEach(() => {
-  destroyAll();
+describe('destroyAll()', () => {
```

Comment:
No, this is wrong. This was intended to be a global `afterEach` hook.

Reply:
Im going to revert it and ignore the rule.